core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -793,7 +793,7 @@ class JsonProtocolSuite extends SparkFunSuite {
   }
 
   test("SPARK-42206: skip logging of all-zero Task Executor Metrics") {
-    val allZeroTaskExecutorMetrics = new ExecutorMetrics(Map.empty[String, Long])
+    val allZeroTaskExecutorMetrics = new ExecutorMetrics(Array.emptyLongArray)
```

Comment:
`Map.empty[String, Long]` should  also ok for Scala 2.13 after https://github.com/apache/spark/pull/39772 merged